### PR TITLE
translate file structure page

### PR DESCRIPTION
**Pull request by umutcanbolat**

### CODE REVIEW

File: content/docs/faq-structure.md [hunk 1]
```diff
@@ -59,16 +59,16 @@ components/
...
 
-#### Don't overthink it {#dont-overthink-it}
+#### Çok fazla düşünmeyin {#dont-overthink-it}
 
-If you're just starting a project, [don't spend more than five minutes](https://en.wikipedia.org/wiki/Analysis_paralysis) on choosing a file structure. Pick any of the above approaches (or come up with your own) and start writing code! You'll likely want to rethink it anyway after you've written some real code.
+Eğer bir projeye henüz yeni başlıyorsanız, dizin yapısı üzerine [beş dakikadan daha fazla zaman harcamayın](https://en.wikipedia.org/wiki/Analysis_paralysis). Yukarıdaki yaklaşımlardan herhangi birini seçin (veya kendi yönteminizi getirin) ve kod yazmaya başlayın! Biraz gerçek kod yazdıktan sonra bu husus hakkında muhtemelen yeniden düşünmek isteyeceksinizdir.
```
Comment (alioguzhan) [inline content/docs/faq-structure.md]: Well, we don't have access to Wikipedia so we can replace the link with https://eksisozluk.com/analysis-paralysis--833201

Comment (umutcanbolat) [reply content/docs/faq-structure.md]: Is this the general solution to Wikipedia links in this project? What if we will have access in the future? @alioguzhan

Comment (alioguzhan) [reply content/docs/faq-structure.md]: Yes. This is our common solution/decision. Once we will have access, we can make the required updates.

**Comment by alioguzhan** [comment]
Thanks for the translation. Merging it...

**Pull request merged**